app/src/main/res/values/strings.xml

```diff
@@ -176,26 +176,26 @@
     <string name="vivo_keypress_workaround">vivo Origin OS keypress workaround</string>
     <string name="invalid_im">Invalid input method configuration file:\n%1$s</string>
     <string name="invalid_table_dict">Invalid dictionary:\n%1$s</string>
-    <string name="exception_table_im">Table input method configuration file (.conf or .conf.in) not found</string>
-    <string name="exception_table">Table dictionary (.dict or .txt) not found</string>
-    <string name="table_im">Table Input Methods</string>
-    <string name="manage_table_im">Manage Table Input Methods</string>
+    <string name="exception_table_im">Configuration file of table-based input method (.conf or .conf.in) not found</string>
+    <string name="exception_table">Mapping table dictionary (.dict or .txt) not found</string>
+    <string name="table_im">Table-based Input Methods</string>
+    <string name="manage_table_im">Manage Table-based Input Methods</string>
     <string name="exception_table_im_filename">The extension of \"%1$s\" does not indicate a table input method package (.zip)</string>
-    <string name="table_already_exists">Table input method configuration file \"%1$s\" already exists</string>
-    <string name="table_file_does_not_exist_title">Table unavailable</string>
-    <string name="table_file_does_not_exist_message">Table dictionary \"%1$s\" does not exist. Please check the dict file or re-import.</string>
-    <string name="import_table">Import Table</string>
+    <string name="table_already_exists">Configuration file of table-based input methods \"%1$s\" already exists</string>
+    <string name="table_file_does_not_exist_title">Character mapping table unavailable</string>
+    <string name="table_file_does_not_exist_message">Mapping table dictionary \"%1$s\" does not exist. Please check the dict file or re-import.</string>
+    <string name="import_table">Import character mapping table</string>
```

Comment:
This string is used on table files (.conf and .txt) selection dialog, thus the title shoud be "Import table-based input method". But it's usually too long to fit in title, I'd rather keep "Import table".

https://github.com/fcitx5-android/fcitx5-android/blob/10ff08f79b7cefbb2cb959e57fe49f89b2edb8b3/app/src/main/java/org/fcitx/fcitx5/android/ui/main/settings/TableInputMethodFragment.kt#L180-L182